pkg/svc/zserver.go

```diff
@@ -2,51 +2,142 @@ package svc
 
 import (
 	"context"
+	"flag"
+	"time"
 
 	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/my-app/pkg/pb"
+	rpb "github.com/my-app/pkg/resp_pb"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
-// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-// ~~~~~~~~~~~~~~~~~~~~~~~~~ A BRIEF DEVELOPMENT GUIDE ~~~~~~~~~~~~~~~~~~~~~~~~~
-//
-// TODO: Extend the MyApp service by defining new RPCs and
-// and message types in the pb/service.proto file. These RPCs and messages
-// compose the API for your service. After modifying the proto schema in
-// pb/service.proto, call "make protobuf" to regenerate the protobuf files.
-//
-// TODO: Create an implementation of the MyApp server
-// interface. This interface is generated by the protobuf compiler and exists
-// inside the pb/service.pb.go file. The "server" struct already provides an
-// implementation of MyApp server interface service, but only
-// for the GetVersion function. You will need to implement any new RPCs you
-// add to your protobuf schema.
-//
-// TODO: Update the GetVersion function when newer versions of your service
-// become available. Feel free to change GetVersion to better-suit how your
-// versioning system, or get rid of it entirely. GetVersion helps make up a
-// simple "starter" example that allows an end-to-end example. It is not
-// required.
-//
-// TODO: Oh yeah, delete this guide when you no longer need it.
-//
-// ~~~~~~~~~~~~~~~~~~~~~~~~~~~ FAREWELL AND GOOD LUCK ~~~~~~~~~~~~~~~~~~~~~~~~~~
-// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
 const (
 	// version is the current version of the service
 	version = "0.0.1"
 )
 
+var (
+	serverAddr = flag.String("addr", "localhost:9099", "The server address in the format of host:port")
```

Comment:
Usually, all the flags should be declared in the same place (`config.go` in our case) - it's more convenient to support the code at least.
It's not a good idea to spread the flags over the application, as well as using the different libs for the same goal in the same application (flags vs viper). Useful link about flags/pflags/viper: https://jayunit100.blogspot.com/2016/09/viper-go-flag-and-pflag-how-they.html

Reply:
Done